nexus/api.go

```diff
@@ -176,3 +203,28 @@ func addContextTimeoutToHTTPHeader(ctx context.Context, httpHeader http.Header)
 	httpHeader.Set(HeaderRequestTimeout, time.Until(deadline).String())
 	return httpHeader
 }
+
+type Link struct {
+	Data []byte `json:"data"`
+	Type string `json:"type"`
+}
+
+var linkBase64Encoding = base64.StdEncoding
```

Comment:
Redunant

Reply:
The goal was to make sure that `EncodeLink` and `DecodeLink` are using the same base64 encoding object (in this case, `base64.StdEncoding`).

Reply:
Yeah, I don't think that justifies this variable though. But if you feel strongly about it, keep it.

Reply:
Removed.